README.md

```diff
@@ -102,9 +102,22 @@ NOTE: OpenShift cluster consumes a lot of resources, you should have at least 18
 
 You should run `gocli` command:
 ```bash
-gocli run okd --prefix okd-4.1.0-rc.0 --background kubevirtci/okd-4.1.0-rc.0@sha256:71afc34d9299aa11313b43a584e7e9d7e2f962279453b53d853a9d3bcb8b3255
+gocli run okd --prefix okd-4.1.0 --background kubevirtci/okd-4.1.0@sha256:e90cb3690753bc053f85537e531f6242e5d31f5504f17a619d8dc4b72e3ba622
 ```
 
+### How to connect to the OKD console
+
+To connect the OKD console you should add once hosts to the `/etc/hosts`
+
+```bash
+127.0.0.1 console-openshift-console.apps.test-1.tt.testing
+127.0.0.1 oauth-openshift.apps.test-1.tt.testing
```

Comment:
console access didn't work for me. 

the console and oauth services are on two cluster ips which both listen to 443. I'm not sure how console can be exposed. 

@cynepco3hahue  did this work for you ?

Reply:
Can you check how looks `docker ps` command?
It worked for me, but only when I specified `ocp-console-port` to `443`.
In general OpenShift haproxy listening on 443, so when you run `https://console-openshift-console.apps.test-1.tt.testing` or `https://oauth-openshift.apps.test-1.tt.testing` it forwards it to OpenShift router, that knows to forward it to the correct service by DNS name.

Reply:
works for me.